src/pdbpp.py

```diff
@@ -68,6 +68,37 @@ def __getattr__(name):
     raise AttributeError
 
 
+def _normalize_path(filename):
```

Comment:
I think this should be done only with the tests in general.

See https://github.com/pdbpp/pdbpp/pull/272/files#diff-6244362351d7c228d95cde366267b10aR39-R46 for a regexp based approach, but we could also adjust anything matching a path in the check function.

Since you have `_normpath_and_escape` already, we should do it there I guess.

Reply:
Ah, yes, I'd forgotten that you already solved that issue.

I've pushed some commits that (a) reverts any changes to the core `pdbpp.py` and (b) uses the regex-style fixes that you implemented.